kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -1006,15 +993,14 @@
     right: 0;
     bottom: 0;
     left: 0;
+    display: flex;
+    justify-content: space-between;
```

Comment:
I dig this move to using flex + space-between justification to spread things evenly. I wonder how many uses of KGrid we have that could be far more easily solved with a couple lines of CSS.

Reply:
Yeees! I love flex for these more specific layouts.